model_analyzer/config/generate/search_parameters.py

```diff
@@ -90,23 +90,14 @@ def _number_of_configurations_for_parameter(
         return number_of_parameter_configs
 
     def _populate_search_parameters(self) -> None:
-        if self._parameters:
-            self._populate_parameters()
-        else:
-            self._populate_default_parameters()
```

Comment:
why did this method get removed?

Reply:
This was just a placeholder until this code got added. The new code has the correct functionality.